packages/google_maps_flutter/ios/Classes/JsonConversions.m

```diff
@@ -32,4 +32,30 @@ + (CGPoint)toPoint:(NSArray*)data {
                      [FLTGoogleMapJsonConversions toDouble:data[1]]);
 }
 
-@end
\ No newline at end of file
++ (NSArray*)positionToJson:(CLLocationCoordinate2D)position {
+  return @[ @(position.latitude), @(position.longitude) ];
+}
+
++ (UIColor*)toColor:(NSArray*)data {
+  NSNumber* numberColor = (NSNumber*)data;
```

Comment:
Just out of curiosity, how does casting an NSArray to an NSNumber work?

Reply:
Not sure either, I just sent an integer from dart code which was then casted to NSArray on Obj-C. Upon searching and testing, found out that NSArray can be casted to NSNumber,